src/hud/debug_hud.rs

```diff
@@ -19,15 +19,17 @@ pub struct DebugHud {
 
     coordinates_last: Point3<f32>,
     coordinates_geometry_buffers: GeometryBuffers<u16>,
+
+    ratio: f32,
```

Comment:
I think I prefer `aspect_ratio`, `ratio` isn't as descriptive (goes for all definitions of `ratio`)